### PR TITLE
Added SONAR_EDITION env. var.

### PR DESCRIPTION
**Pull request by felipecruz91**
Refers to #966

### CODE REVIEW

File: jenkins/ocp-config/deploy/jenkins-master.yml [hunk 1]
```diff
@@ -193,6 +196,8 @@ objects:
                 secretKeyRef:
                   key: auth-token
                   name: sonarqube
+            - name: SONARQUBE_EDITION
```
Comment (felipecruz91) [inline jenkins/ocp-config/deploy/jenkins-master.yml:199]: @renedupont @michaelsauter Note in `ods-core.env` the name of the variable is `SONAR_EDITION` but *within* Jenkins it is `SONARQUBE_EDITION`.

**Comment by michaelsauter** [comment]
This should not be needed. The env var is set in the image: https://github.com/opendevstack/ods-core/blob/master/jenkins/ocp-config/build/bc.yml#L110.

It was done that way as in the first version of ODS, the env var was not part of either DC or image. To add it in a newer version without requiring users to change their DC, it was added in the image.

**Comment by braisvq1996** [comment]
@michaelsauter If I check the current SonarQube edition in the Jenkins pipeline, this is what we get:

image

image

**Comment by braisvq1996** [comment]
It may be because the env value in Jenkins needs to be SONARQUBE_EDITION instead of SONAR_EDITION ?

image

Env values in Jenkins master:

image

**Comment by michaelsauter** [comment]
@braisvq1996 good point. Then this is a bug and we need to align the variable naming. Can you update this PR accordingly?

**Comment by felipecruz91** [comment]
@michaelsauter As @braisvq1996 has identified a workaround that needs to be done in the shared lib, I'll be closing this PR and opening a new one in the shared lib repository.

**Pull request closed without merging**